src/seeds/config.seed.ts

```diff
@@ -0,0 +1,7 @@
+import { configureDatabase } from '@core/db/db.js';
+import { configureEnv } from '@core/env/env.js';
+
+export const configureSeeds = async (envPath?: string) => {
+  await configureDatabase();
+  await configureEnv(envPath || '.env');
```

Comment:
```suggestion
  await configureEnv(envPath ?? '.env');
```

Nit: esse operador é mais próprio pra isso